packages/stylelint-config-base/package.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Ik mis de stylelint dependencies. Kun je daarnaast een stylelint-config-scss package toevoegen welke de base extend. Voor scss hebben we wat losse dependencies geinstalleerd

Reply:
Voor zover ik kan zien zijn alle rules css compatible, dus er is geen los scss package nodig, aangezien `stylelint-scss` toch in `kunstmaan-application` moet blijven staan.

Reply:
Eigenlijk is het niet heel handig als dat moet. Want dan moet je het nog gedeeltelijk per project onderhouden